tools/unix/build_omim.sh

```diff
@@ -66,8 +66,8 @@ if [ "$(uname -s)" == "Darwin" ]; then
   SPEC=macx-clang
   PROCESSES=4
 else
-  SPEC=linux-clang
-  PROCESSES=$(($(nproc) / 2))
+  SPEC=linux-clang-libc++
```

Comment:
Ok, thanks. I wasn't sure whether push --force would work in this setting. This one's fixed now and clean. Branch pull request will follow